pkgs/development/python-modules/pillow/default.nix

```diff
@@ -5,13 +5,13 @@
 
 import ./generic.nix (rec {
   pname = "Pillow";
-  version = "8.0.1";
+  version = "8.1.0";
 
   disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "11c5c6e9b02c9dac08af04f093eb5a2f84857df70a7d4a6a6ad461aca803fb9e";
+    sha256 = "1fnvc07m15fv59bdlkds1q7rl9jrr3cbbn69sfqvzvdpjbknhxl8";
   };
 
   meta = with lib; {
```

Comment:
Unrelated but the license domain does not resolve anymore. I think https://github.com/python-pillow/Pillow/blob/master/LICENSE would fit.

Reply:
> Like PIL, Pillow is licensed under the open source HPND License:

So `licenses.hpnd`.

Reply:
Updated in https://github.com/NixOS/nixpkgs/pull/111655/commits/b5fa81e5c19cd9e8dc720bff95d34105aa77122f.